test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -579,11 +567,8 @@ public async Task GivenATransactionBundle_WhenAnUnsuccessfulPost_ThenTransaction
         private async Task ExecuteAndValidate<T>(Func<Task<FhirResponse<T>>> action, string expectedAction, ResourceType? expectedResourceType, Func<T, string> expectedPathGenerator, HttpStatusCode expectedStatusCode)
             where T : Resource
         {
-            if (!_fixture.IsUsingInProcTestServer)
-            {
-                // This test only works with the in-proc server with customized middleware pipeline.
-                return;
-            }
+            // This test only works with the in-proc server with customized middleware pipeline
+            Skip.If(!_fixture.IsUsingInProcTestServer);
```

Comment:
All the tests that call these three validation methods also need to be marked as Skippable Facts. If Skip is called on a non-Skippable fact it will cause an error.

Reply:
Added with this commit - [5cc34033e](https://github.com/microsoft/fhir-server/pull/2295/commits/5cc34033ea57bfa2d632973b0e0b8963f804f9da)